ui/packages/consul-ui/app/services/repository/metrics.js

```diff
@@ -29,7 +29,7 @@ export default RepositoryService.extend({
       this.error = new Error(`metrics provider not initialized: ${e}`);
       // Show the user the error once for debugging their provider outside UI
       // Dev.
-      console.error(this.error); // eslint-disable-line no-console
+      console.error(this.error);
```

Comment:
I think you are still working on this, but I noticed you dropped an eslint ignore here, guessing it was by accident?